notebooks/data-sources/TestGrid/metrics/persistent_failures_analysis.ipynb

```diff
@@ -17,7 +17,7 @@
   },
```

Comment:
<p>We should update the text to explain what this specific calculation is doing. I think the above is just a general place holder text from the template. </p><p> </p><p>Explain how we are generating the Persistent Failure values. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>